pkg/tso/keyspace_group_manager_test.go

```diff
@@ -735,6 +766,21 @@ func collectAssignedKeyspaceGroupIDs(re *require.Assertions, kgm *KeyspaceGroupM
 	return ids
 }
 
+func collectAllLoadedKeyspaceGroupIDs(kgm *KeyspaceGroupManager) []uint32 {
```

Comment:
move as `func (kgm *KeyspaceGroupManager)collectAllLoadedKeyspaceGroupIDs() []uint32 {`?

Reply:
this is for test only, so it isn't good to move into KeyspaceGroupManager.